p2p/information-element.h

```diff
@@ -90,34 +90,34 @@ typedef struct DeviceinformationBits2_{
     unsigned tdls_persistent_group : 1;
     unsigned tdls_persistent_group_reinvoke : 1;
     unsigned reserved2 : 2;
-} __attribute__ ((packed)) DeviceinformationBits2;
+};
 
-typedef struct DeviceInformationSubelement_{
+struct __attribute__ ((packed)) DeviceInformationSubelement {
     uint8_t id;
     uint16_t length;
     DeviceinformationBits2 field2;
     DeviceinformationBits1 field1;
     uint16_t session_management_control_port;
     uint16_t maximum_throughput;
-} __attribute__ ((packed)) DeviceInformationSubelement;
+};
 
-typedef struct AssociatedBSSIDSubelement_ {
+struct __attribute__ ((packed)) AssociatedBSSIDSubelement {
     uint8_t id;
     uint16_t length;
     uint8_t bssid[6];
-} __attribute__ ((packed)) AssociatedBSSIDSubelement;
+};
 
-typedef struct CoupledSinkStatus_ {
+struct __attribute__ ((packed)) CoupledSinkStatus {
```

Comment:
What's the advantage of a macro? If someone went through the trouble of making that macro platform  (compiler-) independent, then I could see the point but I'm not going to even try that blindly (Why support MSVC if connman isn't used in Windows)...